website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,12 +24,15 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
+                    limit_recipient = rec.env['ir.default'].get('res.config.settings', 'email_recipient_limit')
```

Comment:
@TimLai125 This is a variable to store value for the our setting of limited email recipients. I am not sure whether it is a correct way to do so.